Algorithms/BubbleSort.java

```diff
@@ -0,0 +1,29 @@
+
+public class BubbleSorting{  
```

Comment:
Please do add it in the `sorting` folder, if the sorting folder already has `BubbleSort.java` then you can try with different algorithms.